js/src/behaviors/tabs.ts

```diff
@@ -0,0 +1,120 @@
+/**
```

Comment:
Let's remove this file and reuse this component https://github.com/github/tab-container-element

This will ensure the right aria attributes and keyboard support are available.

Reply:
That looks very nice

Reply:
updated, imported the tab-container-element. Still needed the tabs file for the previous and next button navigation.

Reply:
I believe this has been taken care of.  Resolving.  Please feel free to reopen if necessary.